datagrid/datagrid65-mysql-persistent.json

```diff
@@ -377,13 +377,13 @@
                     }
                 ],
                 "selector": {
-                    "deploymentConfig": "${APPLICATION_NAME}-mysql"
+                    "application": "${APPLICATION_NAME}-mysql"
```

Comment:
We discussed adding a ``component`` label for identifying other deployments in the template.